fix: Pectra activation update (#15968)
ethereum/ethereum-org-website: public/content/roadmap/pectra/index.md

---
title: Prague-Electra (Pectra)
description: Learn about the Pectra protocol upgrade
lang: en
---

# Pectra {#pectra}

> Prague-Electra (Pectra) is an upgrade to the Ethereum network, following [Dencun](/roadmap/dencun/). This major upgrade brings changes to both the execution and consensus layer of Ethereum. The shortened name Pectra is a combination of Prague and Electra, which are the respective names for the execution and consensus layer specification changes. Together, these changes bring a number of improvements to Ethereum users, developers and validators.

Reviewer: Would make this whole first sentence past-tense now.. and maybe simplify a touch:

```suggestion
The Pectra network upgrade followed [Dencun](/roadmap/dencun/) and brought changes to both the execution and consensus layer of Ethereum. The shortened name Pectra is a combination of Prague and Electra, which are the respective names for the execution and consensus layer specification changes. Together, these changes bring a number of improvements to Ethereum users, developers and validators.
```

(We explain the naming breakdown in a dedicated sentence, so not sure it's needed at the start imo)

Author: Agree! Thanks :)
